File: src/main/java/com/hohltier/decompiler/window/menu/file/File.java
package com.hohltier.decompiler.window.menu.file;

import com.hohltier.decompiler.window.utils.Language;
import javax.swing.*;
import java.awt.event.KeyEvent;

public class File extends JMenu {

    public File() {
        setText(Language.get("file"));
        setMnemonic(KeyEvent.VK_F);

        add(new FileOpenFile());
        add(new FileCloseTab());
        add(new JSeparator());
        add(new FileNewInstance());
        add(new FileExit());
    }

}

File: src/main/kotlin/net/pryoscode/decompiler/window/menu/file/File.kt
package net.pryoscode.decompiler.window.menu.file

import net.pryoscode.decompiler.window.menu.file.items.CloseTab
import net.pryoscode.decompiler.window.menu.file.items.Exit
import net.pryoscode.decompiler.window.menu.file.items.NewWindow
import net.pryoscode.decompiler.window.menu.file.items.OpenFile
import java.awt.event.KeyEvent
import javax.swing.JMenu
import javax.swing.JSeparator

class File : JMenu("File") {

    init {
        mnemonic = KeyEvent.VK_F
        add(OpenFile())
        add(CloseTab())
        add(JSeparator())
        add(NewWindow())
        add(Exit())
    }

}